src/univalue/include/univalue_escapes.h

```diff
@@ -1,6 +1,6 @@
 // Automatically generated file. Do not modify.
```

Comment:
Well, then maybe leave a comment along the lines of:

Generator removed in commit `1271737355542049eb9e368174903af379106ffd`

Reply:
I'll just remove the `// Automatically generated file. Do not modify.` comment entirely.